Shared/Samples/Show service areas for multiple facilities/README.metadata.json

```diff
@@ -0,0 +1,31 @@
+{
+    "category": "Routing and Logistics",
```

Comment:
I feel this is the right category, so I'll move https://github.com/Esri/arcgis-maps-sdk-swift-samples/pull/434 to the same category as this sample.